### PR TITLE
W4mclassfilter 0.98.16

### PR DESCRIPTION
**Pull request by eschen42**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

 * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
 * `@BiocondaBot please add label` will add the `please review & merge` label.
 * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.

For members of the Bioconda project, the following command is also available:

 * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.

If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by eschen42** [comment]
@BiocondaBot please add label

File: recipes/w4mclassfilter/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   version: {{ version }}
 
 source:
   url: https://github.com/HegemanLab/w4mclassfilter/releases/download/v{{ version }}/w4mclassfilter_{{ version }}.tar.gz
-  md5: {{ md5 }}
+  sha256: {{ sha256 }}
 
 build:
   noarch: generic
-  number: 0
```
Comment (npavlovikj) [inline recipes/w4mclassfilter/meta.yaml]: Since the version is updated, the build number should be 0, and not 1.

Comment (eschen42) [reply recipes/w4mclassfilter/meta.yaml]: If I change it to zero, it no longer will build: CircleCI said (correctly) that I had already submitted a build with build 0 - I discovered after creating the pull request that I was supposed to use a sha256 hash, but when I resubmitted, CircleCI said that I had already done a build and needed to increment the build number.  

Are you suggesting that if I have a build failure (or success) I have to change my package version? If so (and I want the number to reveal the version of the content), would "v0.98.16-1" be a satisfactory alternative package version name?

**Comment by bgruening** [comment]
This diff looks wired. There is already the same version uploaded. And it also has a sha256 checksum: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/w4mclassfilter/meta.yaml

So I guess we can just close this PR.

**Pull request closed without merging**